app/views/completed_transactions/_fields.html.erb

```diff
@@ -35,10 +35,6 @@
           { checked: (f.object.promotion_choice == "organ_donor"), disabled: @resource.locked_for_edits?, class: 'promotion-choice-organ-donor' } %>
         <%= f.label :promotion_choice, "Promote organ donation", value: 'organ_donor' %>
         <br />
-        <%= f.radio_button :promotion_choice, 'register_to_vote',
-          { checked: (f.object.promotion_choice == "register_to_vote"), disabled: @resource.locked_for_edits?, class: 'promotion-choice-register-to-vote' } %>
```

Comment:
The `promotion-choice-register-to-vote` class isn't needed anymore, would be good to remove that as well from forms.scss

Reply:
Have removed this class from the CSS.